exdb/tests/integration_tests.py

```diff
@@ -187,39 +194,53 @@ def setUp(self):
         self.test_org = self.create_org()
         self.test_keyword = self.create_keyword()
 
+    def get_post_data(self, start, end, name='test', description='test', ex_type=None, sub_type=None,
+                      guest='1', recognition=None, keywords=None, goal='a', action='submit'):
+
+        ex_type = ex_type or self.test_type.pk
+        sub_type = sub_type or self.test_sub_type.pk
```

Comment:
This `get_post_data()` method is inconsistent with the `get_post_data()` method from `ExperienceCreationFormTest` in that it takes integers for `ex_type` and `sub_type` instead of `Type` & `SubType` objects. Either way is fine, but make them consistent with each other.